dts/bindings/sensor/brcm,afbr-s50.yaml

```
@@ -26,6 +26,9 @@ description: |
           compatible = "brcm,afbr-s50";
           reg = <0>;
           spi-max-frequency = <10000000>;
+          measurement-mode = <1>;
+          dual-freq-mode = <0>;
+          odr = <10>;
```

Nit:

`#include <zephyr/dt-bindings/sensor/afbr_s50.h>`

```suggestion
          measurement-mode = <AFBR_S50_DT_MODE_SHORT_RANGE>;
          dual-freq-mode = <AFBR_S50_DT_DFM_MODE_OFF>;
          odr = <10>;
```